sequence_processing_pipeline/Job.py

```diff
@@ -150,6 +150,10 @@ def _system_call(self, cmd, allow_return_codes=[], callback=None):
         :return: a dictionary containing stdout, stderr, and return_code as
                  key/value pairs.
         """
+        # when system-wide binaries such as 'sbatch' and 'sacct' are not
+        # present, assume executable w/no path is present in current
+        # directory.
+        cmd = 'export PATH="$PATH:.";' + cmd
```

Comment:
Why not:

```suggestion
        cmd = './' + cmd
```

Reply:
When the code is running in production or qiita-rc, the executable will be loaded in via linux module and be available on the path. When running in unittests, a mock executable will be created in the current directory. I believe if we simply prepend './' to the cmd it would not locate the binaries in production.

Reply:
I considered it. This was the quickest solution and it stuck. I'm happy to revisit it however.

Reply:
Can't something equivalent to `export PATH=$(pwd):$PATH` be added to github actions and that solves the issue?

Reply:
Probably.